recipes/abseil/all/test_v1_package/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.8)
```

Comment:
The all content of this file can be replaced by:

```cmake
cmake_minimum_required(VERSION 3.1)
project(test_package)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package)
```

Reply:
@SpaceIm 
Thanks your views!
Fixed.

There are one difference between test_package and test_v1_package.
To replace test_v1_package, I have to modify CMakeLists.txt in test_package followings:

```
if(TARGET abseil::abseil)
target_link_libraries(${PROJECT_NAME}_global PRIVATE abseil::abseil)
else()
target_link_libraries(${PROJECT_NAME}_global PRIVATE CONAN_PKG::abseil)
endif()
```

Reply:
Why? Just use abseil::abseil for the global target. It should be defined in both cases.

Reply:
@SpaceIm 
Thanks!
I use absl::absl instead of CONAN_PKG::abseil.